docs/bigtable-row.rst

```diff
@@ -0,0 +1,11 @@
+Bigtable Row
+============
+
+A
```

Comment:
@jonparrott Sorry this is unfinished, I have to be AFK for the afternoon. Before merging this PR, this section should mirror what we have in `bigtable-column-family.rst` as a `RowFilter` reference (it's very similar to `GarbageCollectionRule` as a parent class)